pom.xml

```diff
@@ -21,17 +21,82 @@
     <parent>
         <groupId>org.eclipse.ee4j</groupId>
         <artifactId>project</artifactId>
-        <version>1.0.4</version>
+        <version>1.0.5</version>
     </parent>
 
     <groupId>jakarta.jms</groupId>
     <artifactId>jakarta.jms-api</artifactId>
     <version>2.0.2-SNAPSHOT</version>
-    
+
     <name>jms</name>
     <url>https://projects.eclipse.org/projects/ee4j.jms</url>
 
+    <properties>
+        <!--
+            After every promotion build:
+              - increment spec.build
+            To re-spin a promotion build:
+              - change the project <version> back to the previous value
+              - commit the change
+
+            When the spec turns final:
+              - change spec.non.final to false
+              - change spec.version to the value of spec.newVersion
+              - comment out spec.newVersion
+              - comment out spec.build
+              - update the project <version> to not have -bXX
+        -->
+        <spec.title>Java(TM) Message Service Specification</spec.title>
+        <spec.version>2.0</spec.version>
+        <spec.newVersion>2.1</spec.newVersion>
+        <spec.build>01</spec.build>
+        <spec.impl.version>2.0.2</spec.impl.version>
+        <spec.extension.name>javax.jms</spec.extension.name>
+        <spec.non.final>false</spec.non.final>
+        <spec.api.package>jakarta.jms</spec.api.package>
```

Comment:
Should this not be javax.jms? For this release the package name did not change.

Reply:
It should but then it "generates" javax.jms bundle symbolic name and that leads to a mismatch between file name and symbolic name. For some reason osgi tools (p2, tycho,...) expects these to match. OSGi spec also recommends these to match (with exception of using '_' as a separator between name and version). And in https://wiki.eclipse.org/JakartaEE_Maven_Versioning_Rules you can see, that symbolic name and file name are supposed to match as well.

As soon as the specversion plugin appears in eclipse-ee4j, someone should update it to jakarta rules. As it is now, it is used to define proper version numbers

Reply:
Okay, thanks for the explanation. @m0mus Do we need to do a final check for all artefacts if they confirm to this?